aqua-examples/near-integration/README.md

```diff
@@ -338,80 +451,77 @@ data Result:
+func rpc_foo(network_id: string, node: string, service_id: string) -> string: 
     on node:
-        RPCService service_id
-        res <- RPCService.node_status(network_id, block_ref)
+        NearRpcServices service_id
+        res <- NearRpcServices.node_status(network_id)
         if res.stderr:
             result <<- "call failed"
         else:
             result <<- res.stdout
         <- result
 ```
 
-Before we can use our Fluence NEAR adapter, we need to deploy our Wasm modules to one or more host peers. We can do that with [Aqua CLI](https://doc.fluence.dev/aqua-book/aqua-cli):
+Before we can use our Fluence NEAR adapter, we need to deploy our Wasm modules to one or more host peers. We can do that with [Fluence CLI](https://github.com/fluencelabs/fluence-cli#fluence-deploy):
 
 ```bash
-aqua remote deploy_service \
-     --addr /dns4/kras-04.fluence.dev/tcp/19001/wss/p2p/12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi \
-     --config-path configs/near_deploy_cfg.json \
-     --service near-adapter
+fluence deploy
 ```
 
-Which gives us the deployment information:
+Fluence will make sure that all required services and modules are in place, can be either downloaded or built. It gives us the deployment confirmation:
 
 ```bash
-Your peerId: 12D3KooWNg9YAJWzSRC5Wt2U38beXWAZQfZc2xxqVLVogRkSYKiv
-"Going to upload a module..."
-2022.02.07 01:58:44 [INFO] created ipfs client to /ip4/164.90.164.229/tcp/5001
-2022.02.07 01:58:44 [INFO] connected to ipfs
-2022.02.07 01:58:45 [INFO] file uploaded
-"Going to upload a module..."
-2022.02.07 01:58:45 [INFO] created ipfs client to /ip4/164.90.164.229/tcp/5001
-2022.02.07 01:58:46 [INFO] connected to ipfs
-2022.02.07 01:58:49 [INFO] file uploaded
-"Now time to make a blueprint..."
-"Blueprint id:"
-"24b026f9b1b3f189d7998f875c0eb0c3394e546ee248ea292a27a555d3643774"
-"And your service id is:"
-"b39eb93d-0e7b-478c-976f-2e5b05ec02fb"
-```
-
-Please note the node id, "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi", and service id "b39eb93d-0e7b-478c-976f-2e5b05ec02fb" for future use in our Aqua. Let's have a look at our aqua script in 'aqua/near_adapter_demo.aqua`:
+Making sure all services are downloaded... done
+Making sure all modules are downloaded and built... done
+Going to deploy project described in <path-to-examples>/examples/aqua-examples/near-integration/services/fluence.yaml
+Deploying:
+  service: nearAdapter
+  deployId: default
+  on: 12D3KooWR4cv1a8tv7pps4HH6wePNaK6gf1Hww5wcCMzeWxyNw51
+... done
+Compiling <path-to-examples>/examples/aqua-examples/near-integration/services/.fluence/aqua/deployed.app.aqua... done
+```
+
+
+Please note the helper generated in Aqua by the CLI `deployed.app.aqua` for future use in our Aqua. Let's have a look at our aqua script in `src/aqua/main.aqua`:
 
 ```aqua
--- aqua/near_adapter_demo_aqua
-func node_status(network_id: string, node: string, service_id: string) -> Result: 
-    on node:
-        NearRpcServices service_id
+-- aqua/main.aqua
```

Comment:
i got an empty `src/aqua` dir in root and no `deployed.app.aqua` only a non functioning import

Reply:
The `deployed.app.aqua` is located at `<path-to-examples>/examples/aqua-examples/near-integration/services/.fluence/aqua/deployed.app.aqua`. Made it explicit in the doc, thanks!